content/md/articles/cookbooks/cli_build_projects.md

```diff
@@ -306,6 +323,8 @@ we'll see the test runner output (assuming you don't have any tests yet):
     Ran 0 tests containing 0 assertions.
     0 failures, 0 errors.
 
+**Create `test` build function**
```

Comment:
This reads very awkwardly to me -- I suggest striking "build" from this heading.